File: assignment 4/matrixaddition.cpp
#include <iostream>
using namespace std;
class Matrix
 {
   int r=2, c=4, sum[2][4], i, j;
   int a[2][4] = {{1,5,9,4} , {3,2,8,3}};
   int b[2][4] = {{6,3,8,2} , {1,5,2,9}};
   public :void display()
   {

   cout<<"The first matrix is: "<<endl;
   for(i=0; i<r; ++i) {
      for(j=0; j<c; ++j)
      cout<<a[i][j]<<" ";
      cout<<endl;
   }
   cout<<endl;
   cout<<"The second matrix is: "<<endl;
   for(i=0; i<r; ++i) {
      for(j=0; j<c; ++j)
      cout<<b[i][j]<<" ";
      cout<<endl;
   }
   cout<<endl;
   for(i=0;i<r;++i)
   for(j=0;j<c;++j)
   sum[i][j]=a[i][j]+b[i][j];
   cout<<"Sum of the two matrices is:"<<endl;
   for(i=0; i<r; ++i) {
      for(j=0; j<c; ++j)
      cout<<sum[i][j]<<" ";
      cout<<endl;
   }
   }
};
int main()
{
    Matrix obj;
    obj.display();
}

File: Java assignment(3-4)/Java assignment 4/MatrixAddition.java
import java.util.*;
public class MatrixAddition {
	int n,m,x,y;
	Scanner sc=new Scanner(System.in);
	
	void getDetails()
	{		
		
		System.out.println("Enter the number of rows and columns: ");
		m=sc.nextInt();
		n=sc.nextInt();
		int a[][]=new int[m][n];
		int b[][]=new int[m][n];
		int c[][]=new int[m][n];
		System.out.println("Enter the elements of First  matrix: ");
		for( x=0;x<m;x++)
		{
			for( y=0;y<n;y++)
			{
				a[x][y]=sc.nextInt();
				
					
			}
		System.out.println();

		
		
		}
		
		System.out.println("Enter the elements of second  matrix: ");
		for( x=0;x<m;x++)
		{
			for( y=0;y<n;y++)
			{
				b[x][y]=sc.nextInt();	
					
			}
		System.out.println();

		
		
		}
System.out.println("Your Matrix is : ");
for( x=0;x<m;x++)
{
	for( y=0;y<n;y++)
            {
                System.out.print(a[x][y]+"\t");
                
    			
               
            }
             
            System.out.println();
        }
System.out.print("\n");
for( x=0;x<m;x++)
{
	for( y=0;y<n;y++)
            {
                System.out.print(b[x][y]+"\t");
                
    			
           
            }
             
            System.out.println();
        }
		System.out.println("Sum is: ");
		for(x=0;x<m;x++)
		{
			for(y=0;y<n;y++)
			{
				c[x][y]=a[x][y]+b[x][y];
				
			
		
			System.out.print(c[x][y]+"\t");
			
			}
			
			System.out.println();
			
		}
	
		
		
	}

	public static void main(String[] args) {
		MatrixAddition obj=new MatrixAddition();
		obj.getDetails();
		

	}

}
